### PR TITLE
🏃 Update config image patches and pull policy

### PR DESCRIPTION
**Pull request by maelk**
**What this PR does / why we need it**:
This PR updates the image and pull policy patches for the IPAM controller. It also adds some verbosity to the gofmt job and update BMO.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -343,6 +352,10 @@ release: clean-release  ## Builds and push container images using the latest git
 	# Set the manifest image to the production bucket.
 	MANIFEST_IMG=$(PROD_REGISTRY)/$(IMAGE_NAME) MANIFEST_TAG=$(RELEASE_TAG) \
 		$(MAKE) set-manifest-image
+	# TODO : this is temporarily, as long as we don't have BMO releases, we use compatibility
+	# tags. The "capm3-<rleease-tag>" tag must be created on current BMO master branch
+	MANIFEST_IMG_BMO=$(PROD_REGISTRY)/$(BMO_IMAGE_NAME) MANIFEST_TAG_BMO=capm3-$(RELEASE_TAG) \
+		$(MAKE) set-manifest-image
```
Comment (furkatgofurov7) [inline Makefile:358]: not `set-manifest-image-bmo` ?

File: Makefile [hunk 2]
```diff
@@ -343,6 +352,10 @@ release: clean-release  ## Builds and push container images using the latest git
 	# Set the manifest image to the production bucket.
 	MANIFEST_IMG=$(PROD_REGISTRY)/$(IMAGE_NAME) MANIFEST_TAG=$(RELEASE_TAG) \
 		$(MAKE) set-manifest-image
+	# TODO : this is temporarily, as long as we don't have BMO releases, we use compatibility
+	# tags. The "capm3-<rleease-tag>" tag must be created on current BMO master branch
```
Comment (furkatgofurov7) [inline Makefile:356]: There is a typo in the comment

File: Makefile (hunk 1 above)
Comment (maelk) [reply Makefile:358]: good catch

File: Makefile (hunk 2 above)
Comment (maelk) [reply Makefile]: fixed, thank you

**Comment by maelk** [comment]
This requires https://github.com/metal3-io/metal3-dev-env/pull/346 to be merged to pass the tests

**Comment by jan-est** [comment]
lgtm

**Pull request merged**